packages/dds/tree/src/test/events/events.spec.ts

```diff
@@ -161,7 +162,8 @@ describe("EventEmitter", () => {
 		assert.strictEqual(count, 1);
 	});
 
-	it("allows registrations of event names that are symbols", () => {
+	it("correctly handles registrations of event names that are symbols", () => {
```

Comment:
nit: as a test case name I do not prefer "correctly blah". What is correctly? reject? allow?
If the tests can't say what they are testing for (usually means more test cases to do so), then remove the part that says what the expectation is. So, this could just be something like "registrations of event names that are symbols".
If there is symmetry for strings and symbols, then you should be able to generate cases for strings and symbols using for loop that does a pass for a string and another for a symbol, if you don't want to write the test code twice. (not saying you need to change to do that)